### ISSUE TITLE
A instruction with only one stage "fetch" will not show in the window

### ISSUE THREAD

**Issue by asd055189** [shioyadan/Konata#6]
Hello, I found a problem that Konata will ignore the chart about an "only a fetch stage" instruction.
Is this a bug?
![image](https://user-images.githubusercontent.com/38260090/190392848-c206832d-dcc5-423f-ae86-23360bcf4549.png)
```
O3PipeView:fetch:199000:0x00010100:0:45:c_bnez a5, 12
O3PipeView:decode:0
O3PipeView:rename:0
O3PipeView:dispatch:0
O3PipeView:issue:0
O3PipeView:complete:0
O3PipeView:retire:0:store:0
```

**Comment by shioyadan** [comment]
I have just fixed this problem in version 0.38 and just released it. Please give it a try. This fix may cause problems in a gem5 trace, so if you find a problem, please report it. Thanks!

**Comment by asd055189** [comment]
The bug has been fixed, thanks!
